PayPalWebCheckout/src/main/java/com/paypal/android/checkoutweb/PayPalWebCheckoutFundingSource.kt

```diff
@@ -4,5 +4,18 @@ package com.paypal.android.checkoutweb
  * Enum class to specify the type of funding for an order
  */
 enum class PayPalWebCheckoutFundingSource(val value: String) {
-    CREDIT("credit"), PAY_LATER("paylater"), PAYPAL("paypal")
+    /**
+     * CREDIT will launch the web checkout flow with credit funding selected
```

Comment:
I'm not sure what the wording on this one should be - is this also only for eligible customers? Is there any PayPal documentation on this?

Reply:
I would word it similar to `PAY_LATER`, is it just displays the credit options. But i tried to get some docs but couldnt find any